forge-game/src/main/java/forge/game/keyword/Keyword.java

```diff
@@ -28,6 +28,7 @@ public enum Keyword {
     ASSIST("Assist", SimpleKeyword.class, true, "Another player can pay up to %s of this spell's cost."),
     AURA_SWAP("Aura swap", KeywordWithCost.class, false, "%s: You may exchange this Aura with an Aura card in your hand."),
     AWAKEN("Awaken", KeywordWithCostAndAmount.class, false, "If you cast this spell for %s, also put {%d:+1/+1 counter} on target land you control and it becomes a 0/0 Elemental creature with haste. It's still a land."),
+    BACKUP("Backup", KeywordWithAmountAndType.class, false, "When this creature enters the battlefield, put {%1$d:+1/+1 counter} on target creature. If that's another creature, it gains the following ability until end of turn."),
```

Comment:
shouldn't it be just withAmount or does it mess up the description?

Reply:
Crashes with `KeywordWithAmount` and Backup:2:**THIS**

Reply:
Hmm I don't know If that's okay for now

I would prefer the Type ones only for the ones with types

@tehdiplomat your opinion?

Reply:
Yea I feel like we'd need an extra Keyword class for this maybe `KeywordWithAmountAndOptions` or something like that, where we can just pass "extra things" that might get handled a bit different depending on the keyword.

Reply:
There was already some support for "Extra" on KeywordWithAmount, so I expanded that